scripts/gulpfiles/build_tasks.js

```diff
@@ -342,8 +342,18 @@ function buildDeps(done) {
     'core',
     'blocks',
   ];
+
+  const testRoots = [
+    closurePath,
+    'core',
+    'blocks',
+    'generators',
+    'tests/mocha'
+  ];
   const args = roots.map(root => `--root '${root}' `).join('');
+  const testArgs = testRoots.map(root => `--root '${root}' `).join('');
   execSync(`closure-make-deps ${args} > tests/deps.js`, {stdio: 'inherit'});
+  execSync(`closure-make-deps ${testArgs} > tests/mocha/mocha_deps.js`, {stdio: 'inherit'});
```

Comment:
What do you think about having this be `tests/deps.mocha.js`?  It seems tidier to me to keep all the generated code together, and not intermixed with the hand-written code.

Reply:
Sounds good to me. Done.